### PR TITLE
Add message processor and message store module.md files

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Add message processor and message store module.md files and add the relevant templates to VS Code dashboard.

### CODE REVIEW

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md [hunk 1]
```diff
@@ -1,171 +1,52 @@
...
 
-The WSO2 Store Forward Connector provides a **Message Storing Client** and a **Message Forwarding Processor** to handle messages 
-received at an API endpoint. The Storing Client will store the received messages in a queue in a message broker for later processing, 
-and the Message Processor will listen to the message broker and forward any new messages to the desired HTTP endpoint.
+## Using the Template
 
-Reliable delivery is achieved by removing the message from the queue only after successfully delivering the message. If the receiving 
-endpoint is not available at the moment, the service will retry to send the message. After several retries, if the 
-message is still not delivered, message forwarder will back up the message, and continue with the next message. 
+Run the following command to pull the `message_processor` and `message_store` template from Ballerina Central.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]:
```suggestion
Run the following commands to pull the `message_processor` and `message_store` template from Ballerina Central.
```

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md [hunk 2]
```diff
@@ -1,171 +1,52 @@
-Implements the message processor for reliable delivery tutorial.
+Template for Message Processor Reliable Delivery
 
-# Reliable Message Delivery
+# Service Orchestration
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]: Why are we calling this Service Orchestration?

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md [hunk 3]
```diff
@@ -1,171 +1,52 @@
...
-* geronimo-j2ee-management_1.1_spec-1.0.1.jar
-* hawtbuf-1.11.jar
-
-This example uses ActiveMQ version 5.15.5. You can select the relevant jar files according to the ActiveMQ version you have installed.
-
-Let's create a folder called `lib` under project root path. Then copy above jar files into the lib folder.
-
-Next, open the `Ballerina.toml` file and add the following below `[dependencies]` section. During project build, 
-ActiveMQ jar files will be added to the executable jar.
+Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]:
```suggestion
Now navigate into the above module directory you created and run the following commands to apply the predefined templates you pulled earlier.
```

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md [hunk 4]
```diff
@@ -1,171 +1,52 @@
-Implements the message store for reliable delivery tutorial.
+Template for Message Store Reliable Delivery
 
-# Reliable Message Delivery
+# Service Orchestration
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md]: Is this the correct name?

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md [hunk 5]
```diff
@@ -1,171 +1,52 @@
...
 
-The WSO2 Store Forward Connector provides a **Message Storing Client** and a **Message Forwarding Processor** to handle messages 
-received at an API endpoint. The Storing Client will store the received messages in a queue in a message broker for later processing, 
-and the Message Processor will listen to the message broker and forward any new messages to the desired HTTP endpoint.
+## Using the Template
 
-Reliable delivery is achieved by removing the message from the queue only after successfully delivering the message. If the receiving 
-endpoint is not available at the moment, the service will retry to send the message. After several retries, if the 
-message is still not delivered, message forwarder will back up the message, and continue with the next message. 
+Run the following command to pull the `message_store` and `message_processor` template from Ballerina Central.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md]:
```suggestion
Run the following commands to pull the `message_store` and `message_processor` templates from Ballerina Central.
```

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md [hunk 6]
```diff
@@ -1,171 +1,52 @@
...
 ```bash
-$ ballerina build -a
+$ ./activemq console
 ```
 
-The build command would create two executable jar files for message store and processor modules. 
-
-Now, start the message broker you installed previously. If you are using Active MQ, you can navigate to `<ACTIVE_MQ_HOME>/bin` 
-and start the broker by running the command below.
+Then let’s build the module. Navigate to the project root directory and execute the following command.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]:
```suggestion
Let’s build the module. Navigate to the project root directory and execute the following command.
```

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]: Fixed

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md (hunk 3 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]: Fixed

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md [hunk 7]
```diff
@@ -1,171 +1,52 @@
...
 ```bash
-$ ballerina build -a
+$ ./activemq console
 ```
 
-The build command would create two executable jar files for message store and processor modules. 
-
-Now, start the message broker you installed previously. If you are using Active MQ, you can navigate to `<ACTIVE_MQ_HOME>/bin` 
-and start the broker by running the command below.
+Then let’s build the module. Navigate to the project root directory and execute the following command.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md]:
```suggestion
Let’s build the module. Navigate to the project root directory and execute the following command.
```

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md (hunk 4 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_store/Module.md]: Fixed

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md (hunk 1 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]: Fixed

File: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md (hunk 6 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md]: Fixed

**Pull request merged**